MySQL Helper Script uses always debian repos

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

MySQL

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/mysql.sh)"

### 📝 Provide a clear and concise description of the issue.

When using the [MySQL helper script](https://community-scripts.github.io/ProxmoxVE/scripts?id=mysql) with advanced settings you can choose whether to use Debian or Ubuntu.
In my case I'd want to use Ubuntu 24.04 which is accepted by the script and also successfully installed afterwards.

However during the installation script for MySQL a repository for APT is about to be added:
https://github.com/community-scripts/ProxmoxVE/blob/f7762da69c2ad228e3a3efbff06df4b70ce9ea0b/install/mysql-install.sh#L36
As you can see the url points statically to the debian repository, but reads the correct codename dynamically. In my case this would result in the following:
`deb [signed-by=/usr/share/keyrings/mysql.gpg] http://repo.mysql.com/apt/debian noble`
Since `noble` is a codename for Ubuntu, but not debian, this is about to fail.

Since there is an option to choose between Debian and Ubuntu, the script should be able to handle this preference correctly.

MySQL serves repositories for
- Debian: http://repo.mysql.com/apt/debian/
- Ubuntu: http://repo.mysql.com/apt/ubuntu/

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Ubuntu 24.04

### 🔄 Steps to reproduce the issue.

- Use Advanced Settings
- Set Distribution to Ubuntu
- Set Ubuntu Version to 24.04
- Run through the other settings (not relevant for this issue)

### ❌ Paste the full error output (if available).

It's not the verbosed output, but I traced it back to the used APT repository.

```
Would you like to install the MySQL 8.4 LTS release instead of MySQL 8.0 (bug fix track; EOL April-2026)? <y/N> y
 ⠦ Installing MySQL
[ERROR] in line 50: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.
```

### 🖼️ Additional context (optional).

_No response_

--- a/install/mysql-install.sh
+++ b/install/mysql-install.sh
@@ -33,7 +33,11 @@ fi
 
 msg_info "Installing MySQL"
 curl -fsSL https://repo.mysql.com/RPM-GPG-KEY-mysql-2023 | gpg --dearmor  -o /usr/share/keyrings/mysql.gpg
-echo "deb [signed-by=/usr/share/keyrings/mysql.gpg] http://repo.mysql.com/apt/debian $(lsb_release -sc) ${RELEASE_REPO}" >/etc/apt/sources.list.d/mysql.list
+if [ "$(lsb_release -si)" = "Debian" ]; then
+    echo "deb [signed-by=/usr/share/keyrings/mysql.gpg] http://repo.mysql.com/apt/debian $(lsb_release -sc) ${RELEASE_REPO}" >/etc/apt/sources.list.d/mysql.list
+else
+    echo "deb [signed-by=/usr/share/keyrings/mysql.gpg] http://repo.mysql.com/apt/ubuntu $(lsb_release -sc) ${RELEASE_REPO}" >/etc/apt/sources.list.d/mysql.list
+fi
 $STD apt-get update
 export DEBIAN_FRONTEND=noninteractive
 $STD apt-get install -y \

